NXDOC-2824: OpenSearch2 Audit Documentation (2025) (#2480)
nuxeo/doc.nuxeo.com-content: src/nxdoc/nuxeo-server/data-store/data-modeling/audit.md

Nuxeo documents and events can have a lot of custom properties, so if you want to log some specific events or document properties, the [Extended Info](#extendedinfo) allows for a Key/Value type storage that will be associated to the main `LogEntry` record. These informations are extracted from the event message using and EL (Expression Language) expression and stored into a map.

### Audit Back-ends

> Since LTS 2025, you have to explicitly choose a backend implementation among the following one:

Reviewer: ```suggestion
Since LTS 2025, you have to explicitly choose a backend implementation from the following implementations listed below:
```

| Backend                                  | Marketplace Package     |
| ---------------------------------------- | ----------------------- |

> | In-Memory (default)                      | Built-in Nuxeo Server   |

Reviewer 2: Can we add a "Not for production" somewhere?

Fore more information about the global Elasticsearch setup, see [Elasticsearch Setup]({{page page='elasticsearch-setup'}}).

#### OpenSearch 2.x Back-end

To use OpenSearch 2.x back-end, you have to install the `nuxeo-audit-opensearch2` package.

> The entries will be stored in the `nuxeo-audit` index by default, you can configure it by setting the `nuxeo.audit.backend.default.opensearch2.index.name` property in

Reviewer: ```suggestion
The entries are stored in the `nuxeo-audit` index by default. You can configure it by setting the `nuxeo.audit.backend.default.opensearch2.index.name` property in 
```

your `nuxeo.conf`.

#### MongoDB Back-end

To use MongoDB back-end, you have to install the `nuxeo-audit-mongodb` package.

> The entries will be stored in the `audit` collection by default, you can configure it by setting the `nuxeo.mongodb.audit.collection.name` property in your `nuxeo.conf`.

Reviewer: ```suggestion
The entries are stored in the `audit` collection by default. You can configure it by setting the `nuxeo.mongodb.audit.collection.name` property in your `nuxeo.conf`.
```
